src/ui/components/Events/Event.tsx

```diff
@@ -64,8 +64,8 @@ export default function Event({
       onClick={onClick}
       onKeyDown={onKeyDown}
       className={classNames(
-        "event flex flex-row justify-between items-center space-x-2 user-select-none",
-        "group block p-3 w-full rounded-lg hover:bg-gray-100 focus:outline-none cursor-pointer",
+        "event flex flex-row justify-between items-center mb-1 mt-1 user-select-none",
+        "group block p-1 pl-3 pr-2 w-full rounded-lg hover:bg-gray-100 focus:outline-none cursor-pointer",
```

Comment:
Since `pl-3` handles left and `pr-2` handles right it's probably best to use `py-1` rather than `p-1`. I make this mistake all the time, but technically classes don't cascade depending on the order they are listed, they cascade depending on the order they are declared, so it's best not to have two conflicting tailwind classes, since it's kind of like leaving up to chance which one of them will actually take effect.

I haven't investigated this deeply, but I sort of suspect that tailwind is actually pretty smart about this and tries to order things so that it "just works", because I haven't noticed it being a problem even when I mess it up. Still, it seems like a good thing to be on the lookout for.

I really like tailwind. And, also, I am sort of afraid for the first time I meet a front-end dev who has only ever written tailwind and doesn't understand how CSS works and I have to say "back in my day we wrote stylesheets by hand and you had to craft your own classes and selectors!" And then I'll have to cut the conversation short to go clean my dentures or something.

Reply:
lol to all of that

Totally agree! Thanks for the tip, will change to py-1!